### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by Eogram**
Because you guys are jerks.

### CODE REVIEW

**Comment by Eogram** [comment]
Reject Tim

File: index.js [hunk 1]
```diff
@@ -115,18 +115,20 @@ function determineMessage(event){
         'Oh, are you still here',        
     ];
     const BYE_MESSAGES = [
-        'Autobots! Roll out!',
```
Comment (tim-foley) [inline index.js]: These need to stay in the code.

File: index.js [hunk 2]
```diff
@@ -115,18 +115,20 @@ function determineMessage(event){
...
-        'Until I return I\'m leaving you in command. I know you won\’t let me down',
-        'This universe, no matter how vast will never be big enough for you and I to coexist',
-        'Above all, do not lament my absence, for in my spark, I know that this is not the end, but merely a new beginning. Simply put, another transformation',
-        'It\’s been an honor serving with you',
+        'Lederhosen',
         'Bye',
         'Laters',
         'Peace out',
-        'May the Force be with you',
+        'I\'m out',
```
Comment (edsalisb) [inline index.js]: use a template string for this guy, Tim was running into issues on it

**Comment by edsalisb** [comment]
Conflicts

**Pull request merged**